inspirehep/config.py

```diff
@@ -1419,6 +1419,11 @@ def _(x):
     'physics.acc-ph',
     'physics.ins-det',
 ]
+RECORDS_DEFAULT_FILE_LOCATION_NAME = "records"
+"""Name of default records Location reference."""
+
+RECORDS_DEFAULT_STORAGE_CLASS = "S"
+"""Default storage class for record files."""
```

Comment:
I might be misremembering, but isn't this the default of `invenio-records-files`